### PR TITLE
Updates to QC graph Version 2 and documentation:

### PR DESCRIPTION
**Pull request by nkaplan4250**
Updates in version 2 include:
-The version number is now included into the Graph’s title
-The color range for the plot of %Outside of Broadcast Range was
widened from 0-3% to 0-5%. The new range defines 0 as green, 2% as
yellow, and 5% as red
-The label of for each plot has been moved to the top center of each
plot and displayed over the plotted data
-Minor changes have been made to fonts, size, and position of scales
and keys
Additions were also made to the documentation page for Analog
Digitization, which now includes a description of the QC Graphs
function and an example of the output. The documentation for
Installation and Setup now also includes a note about installing
gnuplot to enable the QC graph functionality.

### CODE REVIEW

File: Resources/Documentation/analog_digitization.md [hunk 1]
```diff
@@ -108,10 +108,25 @@ Next, check to make sure vrecord produced the correct files. These are:
 
 1. The video file itself 
 1. Bmdcapture log (filename_bmdcapture.log)
-1. FFmpeg log (filename_ffmpeg_date_time.log)
-1. FFplay log (filename_ffplay_date_time.log)
+1. FFmpeg log (filename\_ffmpeg\_date_time.log)
+1. FFplay log (filename\_ffplay\_date_time.log)
 1. Frame MD5 file (filename.framemd5) if you chose to record frame MD5s, which you should do
-1. A capture options log (filename_capture_options.log)
+1. A capture options log (filename\_capture_options.log)
```
Comment (dericed) [inline Resources/Documentation/analog_digitization.md]: one underscore is escaped but not the other

File: Resources/Documentation/analog_digitization.md [hunk 2]
```diff
@@ -108,10 +108,25 @@ Next, check to make sure vrecord produced the correct files. These are:
...
 1. A QCTools XML file, if you chose to create it (filename.qctools.xml.gz)
+1. An image file of QC graphs (filename\_QC\_output_graphs.jpeg) if you chose to create the QCTools XML file
```
Comment (dericed) [inline Resources/Documentation/analog_digitization.md]: missing an escape

File: Resources/Documentation/analog_digitization.md [hunk 3]
```diff
@@ -108,10 +108,25 @@ Next, check to make sure vrecord produced the correct files. These are:
...
 
 Scroll through the video file to make sure it is complete; in other words, you've captured the entire tape. If all these files exist and the video file looks complete you can move on to further QC.
+
+###QC Graphs
```
Comment (retokromer) [inline Resources/Documentation/analog_digitization.md]: Space missing between `###` and `QG Graphs`

**Comment by nkaplan4250** [comment]
Thanks. I have fixed those errors

**Pull request merged**